tools/proofers/images_index.php

```diff
@@ -79,13 +44,38 @@
 }
 else
 {
+    $image_index_str = _('Image Index');
+
+    output_header("$image_index_str: {$project->nameofwork}");
+
+    echo "
+        <h1>" . html_safe($project->nameofwork) . "</h1>
```

Comment:
Did we ever remerge my changes around sanitizing nameofwork throughout the project?

Reply:
Yes, that was merged into the Unicode branch before it merged into master.